### DISCUSSION TITLE
[Esame 13 Febbraio 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#352] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f9754979-3a1b-437e-a07f-12aa36c1118c)

**Comment by Traster-uni** [comment]
![soluz_es4_esamefebbraio_ProfBandiera](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84441861/a39866db-6fae-423c-a75d-3f33dcbacf10)
